synapse/events/utils.py

```diff
@@ -515,11 +515,13 @@ def _inject_bundled_aggregations(
                     thread.latest_event, serialized_latest_event, thread.latest_edit
                 )
 
-            serialized_aggregations[RelationTypes.THREAD] = {
+            thread_summary = {
                 "latest_event": serialized_latest_event,
                 "count": thread.count,
                 "current_user_participated": thread.current_user_participated,
             }
+            serialized_aggregations[RelationTypes.THREAD] = thread_summary
+            serialized_aggregations[RelationTypes.UNSTABLE_THREAD] = thread_summary
```

Comment:
could this be guarded by the feature flag?

Reply:
Good catch! 👍 Guess my commits weren't quite as atomic as I thought.